AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>for this whole lesson has this already been switched to what we can do not just how? For exampel</p><p> </p><ul><li>Figure out our conversion value with PyMC!</li><li>Douoble check we can trust the results</li><li>State our assumptions before seeing data in hte modell</li><li>Estimate the conversion rate</li><li>Simulate the future</li></ul><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
<p>Done</p>